test/spmd/tensor/parallel/test_2d_parallel.py

```diff
@@ -108,10 +101,10 @@ def shard_module(m, pg):
     m.net2 = _aggregate_local_tensor(m.net2)
 
 
-def _shard_wrap_module(module, module_shard, fsdp_wrap, tp_pg, fsdp_pg):
+def _shard_wrap_module(module, module_shard, fsdp_wrap, twod_mesh, fsdp_pg):
```

Comment:
nit: 2d_mesh?

Reply:
Looks this is invalid syntax for python... maybe let's call it mesh_2d.